src/python/gem5/components/boards/lupv_board.py

```diff
@@ -89,36 +92,66 @@ def __init__(
             raise EnvironmentError("RiscvBoard is not compatible with Ruby")
         self.workload = RiscvLinux()
 
+        excep_code = { 'INT_SOFT_SUPER': 1, 'INT_TIMER_SUPER': 5,
+                        'INT_TIMER_MACHINE': 7, 'INT_EXT_SUPER': 9 }
```

Comment:
The indentation every time you have a statement on multiple lines is off in this file.